src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -156,7 +199,13 @@ public class PullRequestHandler : IHttpHandler
         return success;
     }
 
-    private bool SyncToPr(uint pr, StringBuilder executionResult)
+    private string GetRepoPath(uint repoId)
```

Comment:
If we allow an arbitrary string, we will have to start parsing it for special characters such as .. / \ > < (etc) to prevent injections and whatnot. While I _think_ the current code is hardened against such attacks, I'd rather limit inputs to as narrow a subset as possible to shut the door as much as possible 

Our current design calls for numeric 'id' s, so I would rather leave this to be a uint. Let us solve that problem when we make an explicit design decision to open this up to have non-numeric ids.